proto/subscription/tx.proto

```diff
@@ -22,4 +23,17 @@ message MsgBuy {
 message MsgBuyResponse {
 }
 
+message MsgAddProject {
+  string creator = 1;
+  string consumer = 2;
```

Comment:
the msg should have only:
creator
project (this includes all the keys and policies

Reply:
done